substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/SubstrateExitHandlerFeature.java

```diff
@@ -34,7 +34,7 @@
 public class SubstrateExitHandlerFeature implements InternalFeature {
     @Override
     public void beforeAnalysis(BeforeAnalysisAccess access) {
-        if (SubstrateOptions.InstallExitHandlers.getValue()) {
+        if (SubstrateOptions.InstallExitHandlers.getValue() || VMInspectionOptions.hasJfrSupport() || VMInspectionOptions.hasNativeMemoryTrackingSupport()) {
```

Comment:
Typically, we let options turn on/off other options, so here `EnableMonitoringFeatures` would turn on `InstallExitHandlers` if JFR or NMT is turned on. This is arguably easier, but let me take another look.

Reply:
Should I adopt your suggestion and use `OptionKey.onValueUpdate()` to turn on `InstallExitHandlers`? Or should I leave it as it currently is?

Reply:
I'm currently trying to find out whether we could/should enable `InstallExitHandlers` by default. In any case, I can make the change if any is need.